src/domain/event/EventIsEnrolled.tsx

```diff
@@ -54,13 +57,28 @@ const EventIsEnrolled = () => {
         )}
       </div>
       <h2>{t('event.cancellation.heading')}</h2>
+      <p id="eventCancellationDescription">
+        {disableUnenrolling
+          ? t('enrollment.cancellation.disabledDescription', {
+              unerolHoursBeforeOccurrence: 48,
+            })
+          : t('enrollment.cancellation.enabledDescription', {
+              unerolHoursBeforeOccurrence: 48,
```

Comment:
Shouldn't the AppConfig.enrolmentCancellationTimeLimitHours be used here? E.g.:
```suggestion
              unerolHoursBeforeOccurrence,
            })
          : t('enrollment.cancellation.enabledDescription', {
              unerolHoursBeforeOccurrence,
```
with earlier:
```typescript
  const unerolHoursBeforeOccurrence =
    AppConfig.enrolmentCancellationTimeLimitHours;
```
and `import AppConfig from '../app/AppConfig';`